pkg/cmd/infra/router/f5.go

```diff
@@ -225,7 +225,8 @@ func (o *F5RouterOptions) Run() error {
 
 	factory := o.RouterSelection.NewFactory(oc, kc)
 	watchNodes := (len(o.InternalAddress) != 0 && len(o.VxlanGateway) != 0)
-	controller := factory.Create(plugin, watchNodes)
+	// TODO should ingress be configurable for f5?
```

Comment:
Ok, I'll enable it.  The f5 and template router define their flags separately, so f5 will use ``--f5-enable-ingress``